types/node/net.d.ts

```diff
@@ -261,6 +261,13 @@ declare module 'net' {
          * @since v6.1.0
          */
         readonly connecting: boolean;
+        
+        /**
+         * This is `true` if the socket is not connected yet, either because `.connect()` 
```

Comment:
There is trailing whitespace here, see https://github.com/DefinitelyTyped/DefinitelyTyped/pull/63814/checks